### PR TITLE
Abdul/active-offers

### PR DESCRIPTION
**Pull request by imabdulbasit**
```
nft(address : " ") {
  offers {
     wallet
     price
     tradeStateAddress
     tradeStateBump
     activatedAt
  }
}
```

### CODE REVIEW

File: crates/graphql/src/schema.rs [hunk 1]
```diff
@@ -680,6 +684,75 @@ impl<'a> From<models::AuctionHouse<'a>> for AuctionHouse {
...
+        self.token_mint.clone()
+    }
+    pub fn price(&self) -> UnsignedInt8 {
+        self.price
+    }
+    pub fn trade_state_bump(&self) -> i32 {
+        self.trade_state_bump
+    }
+
+    pub fn activated_at(&self) -> Option<UnsignedInt8> {
```
Comment (kespinola) [inline crates/graphql/src/schema.rs:720]: Can we convert these to datetime utc?

Same with closed_at

Comment (kespinola) [reply crates/graphql/src/schema.rs:720]: May be best to do this conversion when saving to the db. use datatime col type in postgres.


Then we can add a custom scaler for DateTime if not supported natively by juniper.

File: crates/graphql/src/schema.rs [hunk 2]
```diff
@@ -27,18 +27,18 @@ struct Creator {
 }
 
 #[derive(Debug, Clone, Copy)]
-pub struct Lamports(u64);
+pub struct UnsignedInt8(u64);
 
-#[juniper::graphql_scalar(description = "Lamports")]
-impl<S> GraphQLScalar for Lamports
+#[juniper::graphql_scalar(description = "UnsignedInt8")]
+impl<S> GraphQLScalar for UnsignedInt8
```
Comment (kespinola) [inline crates/graphql/src/schema.rs:33]: If we switch activated_at to date time scaler can this stay as a lamports scalar.

Comment (imabdulbasit) [reply crates/graphql/src/schema.rs:33]: yes

**Pull request merged**